### PR TITLE
Add Backend Setup

### PR DESCRIPTION
**Pull request by orwellian225**
Add logging
Add config
Add rocket (http api crate)

### CODE REVIEW

**Review by Konstantinos1504 — COMMENTED** [review]
We change the MIT license in a future date, definetly

File: src/endpoints/tests.rs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+use super::super::rocket;
+use rocket::local::blocking::Client;
+use rocket::http::Status;
+
+#[test]
+fn test_index() {
```
Comment (Konstantinos1504) [inline src/endpoints/tests.rs]: Change name to Tamlin's convention
```rust
fn test_index_none_ok() {
```

**Review by Konstantinos1504 — CHANGES_REQUESTED** [review]

File: src/endpoints/tests.rs (hunk 1 above)
Comment (orwellian225) [reply src/endpoints/tests.rs]: will do

**Review by Zach-schwark — APPROVED** [review]

**Review by Konstantinos1504 — APPROVED** [review]
good

**Review by Zach-schwark — APPROVED** [review]

**Pull request merged**